xlive/H2Tweaks.cpp

```diff
@@ -198,7 +198,7 @@ void DeinitH2Tweaks() {
 }
 
 void setSens(short input_type, float sens) {
-	if (input_type) { //controller
+	if (input_type == 1) { //controller
```

Comment:
I asked how many inputs, cause if its just 1 or 0, you just if input == 1 and else for the 0, if not, you don't need to update this really - although its a tad more readable